src/rules/no-named-default.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  meta: {
+    docs: {},
+  },
+
+  create: function (context) {
+    return {
+      'ImportDeclaration': function (node) {
+        if (node.source == null) return
+
+        node.specifiers.forEach(function (im) {
+          if (im.type === 'ImportSpecifier' && im.imported.name === 'default') {
+            context.report({
+              node: im.local,
+              message: 'Use default import syntax to ' +
```

Comment:
You could use template literal syntax here by the way, makes it a bit simpler (and I think you're allowed to put this all on one line, but let me know if you aren't)
```js
`Use default import syntax to import '${im.local.name}'`
```